extensions/functions_arithmetic.yaml

```diff
@@ -742,6 +742,105 @@ aggregate_functions:
           - value: fp64
         nullability: DECLARED_OUTPUT
         return: fp64?
+  - name: "median"
+    description: Calculate the median for a set of values.
```

Comment:
IMO this should include something like "Returns null if applied to zero records. For the integer implementations, the rounding option determines how the median should be rounded if it ends up midway between two values. For the floating point implementations, they specify the usual floating point rounding mode."

Reply:
You mean the `description`?

Reply:
Yes.